packages/db-ra-data-rest/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@platformatic/db-ra-data-rest",
+  "version": "0.4.0",
+  "description": "React Admin data provider REST API for Platformatic DB",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/cjs/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "*.md",
+    "dist",
+    "src"
+  ],
+  "authors": [
+    "Giacomo Cerquone"
+  ],
+  "license": "MIT",
```

Comment:
This package should be licensed as Apache-2.0 as the rest of them.

Reply:
The `db-ra-data-rest` folder will be removed from this PR because we wait for the PR by @giacomocerquone.
I temporarly added the folder to be able to move forward on my PR.